npm/xpath/src/index.d.ts

```diff
@@ -1,6 +1,7 @@
 /// <reference types="cypress" />
 
 declare namespace Cypress {
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
I'm not totally sure what the right thing to do here is, I made this change to fix the lint error from `no-unused-vars` but this doesn't look exactly right. Should `E` on line 14 extend `Subject` instead of `HTMLElement`? @jordanpowell88 any ideas?

Reply:
Should `Chainable` have `Subject`? Should it just be

```
declare namespace Cypress {
  interface Chainable {
    /**
    * Get one or more DOM elements by an XPath selector.
    * **Note:** you can test XPath expressions from DevTools console using $x(...) function, for example $x('//div') to find all divs.
    * @see https://github.com/cypress-io/cypress-xpath
    * @example
    * cy.xpath(`//ul[@class="todo-list"]//li`)
    *   .should('have.length', 3)
    */
    xpath<E extends Node = HTMLElement>(selector: string, options?: Partial<Loggable & Timeoutable>): Chainable<JQuery<E>>
  }
}
```

Reply:
We have some examples in the docs that don't pass `Subject` to `Chainable`: https://docs.cypress.io/guides/tooling/typescript-support#Types-for-Custom-Commands